canonical-web-and-design/www.ubuntu.com
# 5154 implement design topic page

## Done

- Added design topic page, to match https://blog.ubuntu.com/topics/design
- Moved repeated posts markup into its own template

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/blog/topics/design](http://0.0.0.0:8001/blog/topics/design)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure the page looks and behaves similarly to https://blog.ubuntu.com/topics/design

## Issue / Card

Fixes #5154
